packages/web-frontend/cypress/scripts/generateConfig/generateOverlayConfig.js

```diff
@@ -19,6 +19,8 @@ const urlSchema = buildUrlSchema({
     id: oneOrArrayOf(yup.string().required()),
     project: yup.string().required(),
     orgUnit: yup.string().required(),
+    startDate: yup.string(),
+    endDate: yup.string(),
```

Comment:
Does this need `overlayPeriod` as well?

Reply:
Good observation! Made me realise I should also extract `overalyPeriod` when parsing string urls to objects (see `parseUrl()`) 👀 

The reason I don't include it in the schema is to "discourage" people from directly specifying `overlayPeriod` in their configs (they should use `startDate`/`endDate` instead)... We do the same in report config parsing. But truth is people can still pass this in, if they want (no error is thrown) 😀 

So in essence, `overlayPeriod` should just be used by local objects created by the script, to avoid doing an unnecessary conversion from `url?overlayPeriod=period` => `{ startDate, endDate }` => `url?overlayPeriod`